contexts/credentials/issuer-dependent/v2

```diff
@@ -0,0 +1,5 @@
+{
+  "@context": {
+    "@vocab": "https://www.w3.org/ns/credentials/issuer-dependent#"
```

Comment:
```suggestion
    "@vocab": "https://www.w3.org/ns/credentials/private-use-vocab#"
```

Reply:
+1 to a variation of this URL, other options include:

* private-term
* undefined-term
* local-term
* local-vocabulary

We need to do a better job signalling that 1) usage of the context will inevitably lead to conflicts at scale, and 2) usage of the context is meant to be for local ecosystems only (not global ones).

Reply:
`private-term` seems clear

Reply:
```suggestion
    "@vocab": "https://www.w3.org/ns/credentials/undefined-term#"
```